src/pages/[...slug].tsx

```diff
@@ -42,14 +42,18 @@ interface Props {
 }
 
 export const getStaticPaths: GetStaticPaths = () => {
-  const contentFiles = getContent(["slug", "content"])
+  const contentFiles = getContent("/", ["slug"]).filter(
```

Comment:
why was `"content"` param removed from `getContent`? (see previous version, `getContent(["slug", "content"])`)

Reply:
`content` is not used in this step, you only need the `slug`. The content is relevant in `getStaticProps` when the content is serialized.

Reply:
@nhsz Was there a reason you had this in here? Similarly I wasn't sure why it was there... seems like `slug` is the only thing we use here.